.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,23 +1,32 @@
-> ⚠️ We have stopped merging PRs for now to the Strapi core.<br><br>
-> The reason is that we are developing new architecture for the admin panel and for the plugins.<br>
-> This new architecture will provide stability of the Strapi core as we approach the release of Beta.<br>
-> We appreciate and welcome all your contributions, but until further notice, please do not submit a PR as it will not be merged.<br>
-> Furthermore, you will have to rewrite it based on the new architecture.
+<!--
+⚠️ We have stopped merging PRs for now to the Strapi core.
+The reason is that we are developing new architecture for the admin panel and for the plugins.
+This new architecture will provide stability of the Strapi core as we approach the release of Beta.
+We appreciate and welcome all your contributions, but until further notice, please do not submit a PR as it will not be merged.
+Furthermore, you will have to rewrite it based on the new architecture.
```

Comment:
Furthermore, your pull request will have to be rewritten for the new architecture.